exporter/opencensusexporter/opencensusexporter_test.go

```diff
@@ -22,11 +22,12 @@ import (
 
 func TestOpenCensusTraceExportersFromViper(t *testing.T) {
 	v := viper.New()
-	v.Set("opencensus", struct{}{})
+	v.Set("opencensus", map[interface{}]interface{}{})
```

Comment:
q.: what went wrong with the old code after the change? this was the reason to add the `endpoint` too?

Reply:
After this PR the change Viper expects "opencensus" to be a `map` rather than a `struct`. If we set it to a struct here we will get a different error saying "Decoding failed, expect map got struct".